layouts/partials/js/dropdown.js

```diff
@@ -0,0 +1,87 @@
+/**
+  Toggles the necessary aria- attributes' values on the menus
+  and handles to show or hide them.
+  @param {HTMLElement} element The menu link or button.
+  @param {Boolean} show Whether to show or hide the menu.
+  @param {Number} top Top offset in pixels where to show the menu.
+*/
+function toggleMenu(element, show, top) {
+  let target = document.getElementById(element.getAttribute("aria-controls"))
+
+  if (target) {
+    element.setAttribute("aria-expanded", show)
+    target.setAttribute("aria-hidden", !show)
+
+    if (typeof top !== "undefined") {
+      target.style.top = top + "px"
+    }
+
+    if (show) {
+      target.focus()
+    }
+  }
+}
+
+/**
+  Attaches event listeners for the menu toggle open and close click events.
+  @param {HTMLElement} menu The menu container element.
+*/
+
+function setupContextualMenu(menu) {
```

Comment:
Can you convert the functions in this file to ES6 style "fat-arrow" functions like the other file please? :)

Reply:
I knew I forgot some :) Done!